(LD-468) Update Readme.md (#41)
bitovi/react-native-pmo: README.md

While using the emulator if you wish to use the element inspector, run the `adb shell input keyevent 82` in the terminal.

### Running the local place-my-order-api server for the android emulator

By default, the Android emulator won't be able to access your localhost server.
In order to connect it to the emulator run the `adb reverse tcp:7070 tcp:7070` command. 
Make sure if you're running the place-my-order-api on a different port to replace the `7070` with `port` 
e.g. if you're running on port 3030, run `adb reverse tcp:3030 tcp:3030`

### Cleaning build cache

> While developing or making changes to the `.env` the emulator will used cached results, so you may not see your changes.

Reviewer: ```suggestion
While developing or making changes to the `.env` the emulator will use cached results, so you may not see your changes.
```

> To amend this shut off the emulator and the use the `npm run clean` command. After starting up the emulator should take the new `.env` variables.

Reviewer: ```suggestion
To amend this, shut off the emulator and use the `npm run clean` command. After starting up the emulator should take the new `.env` variables.
```

### Common Emulator Issues

> When using the `npm run android` command, if an existing instance of the emulator is running it will add a message that says so. Be sure to make sure the terminal window that was hosting the React Native dev server isn't still on. 

Reviewer: ```suggestion
When using the `npm run android` command, if an existing instance of the emulator is running it will add a message that says so. Be sure the terminal window that was hosting the React Native dev server isn't still on. 
```

> When turning off the emulator, be sure to `Ctrl+C` to stop the server from running before closing the terminal window. If you're struggling to run the emulator because there is another instance running, there is a chance you may have closed the terminal before stopping the server. In this case, you may have to use Task Manager or Activity Monitor to close the rogue instance of the server. Finally, if none of the proposed solutions work, just trying restarting your work system.

Reviewer: ```suggestion
When turning off the emulator, use `Ctrl+C` to stop the server from running before closing the terminal window. If you're struggling to run the emulator because there is another instance running, there is a chance you may have closed the terminal before stopping the server. In this case, you may have to use Task Manager or Activity Monitor to close the rogue instance of the server. Finally, if none of the proposed solutions work, just try restarting your work system.
```
